### ISSUE TITLE
[Feature] Flatpak Packaging

### ISSUE THREAD

**Issue by jonahgeorge** [WowUp/WowUp#528]
First of all, thanks a ton for creating WowUp! This issue is part feature request, part question regarding whether you would accept contributions adding this.

**Is your feature request related to a problem? Please describe.**

Currently WowUp is packaged on Linux using the AppImage format. This format is great for cross-distribution distributables; however, has a not-so-great user experience for installation and integration into various desktops. Many Gnome-based Linux flavors are integrating Flatpak into their app centers and it would be awesome to be able to manage WowUp via the same method as other software on my system.

**Describe the solution you'd like**

Add WowUp to [Flathub](https://github.com/flathub/flathub/wiki/App-Submission) and make any CI changes necessary to support this new package format.

**Describe alternatives you've considered**

The current Linux distribution!

**Additional context**

- https://docs.flatpak.org/en/latest/electron.html
- https://github.com/electron-userland/electron-builder/issues/512

**Comment by jliddev** [comment]
If someone wants to take this on to get it setup I don't mind but this would be fairly low priority for me.

**Comment by burgurpun** [comment]
#1546

**Comment by p3lim** [comment]
I grew tired of waiting so I created an unofficial repository with Flatpak builds: <https://github.com/p3lim/wowup-flatpak>

Once an official one is in place I'll remove it.
